bot/exts/utils/reminders.py

```diff
@@ -538,17 +571,30 @@ async def _can_modify(self, ctx: Context, reminder_id: str | int, send_on_denial
                     return False
             raise e
 
-        if await has_any_role_check(ctx, Roles.admins):
+        if api_response["author"] == ctx.author.id:
+            log.debug(f"{ctx.author} is the reminder author and passes the check.")
             return True
 
-        if api_response["author"] != ctx.author.id:
-            log.debug(f"{ctx.author} is not the reminder author and does not pass the check.")
-            if send_on_denial:
-                await send_denial(ctx, "You can't modify reminders of other users!")
-            return False
+        if await has_any_role_check(ctx, Roles.admins):
+            log.debug(f"{ctx.author} is an admin, asking for confirmation to modify the reminder.")
 
-        log.debug(f"{ctx.author} is the reminder author and passes the check.")
-        return True
+            confirmation_view = ModifyConfirmationView(ctx.author)
+            await ctx.send(
+                "Are you sure you want to modify someone else's reminder?",
```

Comment:
Maybe mentioning who that person is would be useful?

Reply:
I think replying to the original message that invoked the command would be better.

Reply:
I actually meant the owner of the reminder.

So a reply + mentioning the actual author would be a good combo

Reply:
I see, it has been added now.